Jenkinsfile

```diff
@@ -173,6 +173,7 @@ def RunIntegrationTest(k, v) {
                     unstash 'repo-amd64'
                     echo "running tests on k8s version ${v}"
                     sh 'tests/scripts/makeTestImages.sh load amd64'
+                    sh "tests/scripts/kubeadm.sh clean"
```

Comment:
do we also need cleanup for the localPathPV.sh? The latest build is failing on the local path setup.

Reply:
I think the issue is something else. The image was not even present there. Reorganized the commands.Let's see how the tests go.